src/plugins/intel_cpu/src/config.h

```diff
@@ -55,6 +55,7 @@ struct Config {
     std::string dumpToDot = {};
     std::string device_id = {};
     float fcSparseWeiDecompressionRate = 1.0f;
+    size_t fcDynamicQuantizationGroupSize = 32;
```

Comment:
Currently all decompression/dynamic_quantization optimizations are applied only on x64. So this flag not impacts ARM. Given that I would prefer to avoid adding any conditions there.